hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -1425,6 +1425,13 @@ protected void update(final AssignmentManager am) {
       this.statTimestamp = EnvironmentEdgeManager.currentTime();
       update(regionStates.getRegionsStateInTransition(), statTimestamp);
       update(regionStates.getRegionFailedOpen(), statTimestamp);
+
+      if (ritsOverThreshold != null && !ritsOverThreshold.isEmpty()) {
+        LOG.trace("RIT hashes and states: " +
```

Comment:
Updating logs here looks better. Just small nit: instead of appending string, can we use placeholder `{}` here?
`LOG.trace("RIT hashes and states: {}", val);`